app/views/capture_email.scala.html

```diff
@@ -0,0 +1,51 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.play.views.html.helpers.{form, errorSummary, input}
+@import config.AppConfig
+@import forms.EmailForm._
+
+@(emailForm: Form[String])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(Messages("captureEmail.title")) {
+
+  @errorSummary(messages("common.error.heading"), emailForm, forceDivToAppear = false)
+
+  <h1 id="main-heading" class="form-title heading-xlarge">@messages("captureEmail.title")</h1>
+
+  @form(action = controllers.routes.CaptureEmailController.submit) {
+    <fieldset>
+      <legend class="visuallyhidden">@messages("captureEmail.title")</legend>
+      <div class="form-group">
+        @input(
+          emailForm("email"),
+          '_divClass       -> "form-label",
+          '_labelId        -> "label-email",
+          '_hintId         -> "label-email-hint",
+          '_label          -> messages("captureEmail.title"),
+          '_labelTextClass -> "visuallyhidden",
+          '_inputClass     -> "input--fullwidth",
+          '_maxlength      -> maxLength,
+          '_inputHint      -> messages("captureEmail.hint")
+        )
+      </div>
+    </fieldset>
+
+    <div class="form-group">
+      <button class="btn button" type="submit" id="next">@Messages("common.continue")</button>
```

Comment:
Can you remove the `btn` css please. I don't believe it's adding anything. 
Can you use `messages` instead of `Messages`
Can you remove the id please